plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -414,11 +436,10 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
          auto& chain = chain_plug->chain();
 
          if( chain.is_building_block() ) {
-            _account_fails.report( _idle_trx_time );
+            _time_tracker.report_and_clear( _idle_trx_time );
```

Comment:
It is ok to clear here, but not needed. So I think we can separate `report()` from `clear()` for `time_tracker`.